tests/django_app/manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!../../env/bin/python
```

Comment:
possibly not for this PR but wouldn't it be better to move the django_app outside of the tests folder?